sunkit_image/conftest.py

```diff
@@ -84,3 +85,9 @@ def pytest_runtest_setup(item):
     if isinstance(item, pytest.Function):
         if "remote_data" in item.keywords and not HAVE_REMOTEDATA:
             pytest.skip("skipping remotedata tests as pytest-remotedata is not installed")
+
+
+@pytest.mark.remote_data
+@pytest.fixture(scope="session")
+def test_granule_map():
+    return sunpy.map.Map("https://github.com/sunpy/data/raw/main/sunkit-image/granule_testdata.fits")
```

Comment:
This works but its not a fast download.

Reply:
Pytest handles it automatically.

Reply:
We could, how it interacts with pytest would be interesting.

I can work on that after we are ready with the rest of this PR.

The other choice is that we downsample this image (to be 100 by 100 pixels) and add it to the repo.

Reply:
Ok that's fine with me. I'm fine with this being a separate PR as well.